tests/PHPStan/Rules/DeadCode/NoopRuleTest.php

```diff
@@ -6,6 +6,9 @@
 use PHPStan\Rules\Rule;
 use PHPStan\Testing\RuleTestCase;
 
+/**
+ * @extends \PHPStan\Testing\RuleTestCase<NoopRule>
```

Comment:
Should this not be FQN rule class?

Reply:
It is, the test and the rule class are in the same namespace.

Reply:
Right, of course, by bad. 👍